source/client/process_impl.cc

```diff
@@ -425,6 +425,13 @@ absl::StatusOr<ProcessPtr> ProcessImpl::CreateProcessImpl(
                                                        std::move(typed_dns_resolver_config),
                                                        process_wide));
 
+  if (options.userDefinedOutputPluginConfigs().size() > 0) {
```

Comment:
nit: empty() is slightly preferred over size() http://g3doc/devtools/cymbal/clang_tidy/g3doc/checks/readability-container-size-empty

Reply:
I see and agreed.